content/en/docs/tasks/run-application/access-api-from-pod.md

```diff
@@ -107,9 +110,7 @@ The output will be similar to this:
 ```json
 {
   "kind": "APIVersions",
-  "versions": [
-    "v1"
-  ],
+  "versions": ["v1"],
```

Comment:
You don't have to change this.
The existing coding style may be preferred by some people because it looks clearer.